packages/flutter/lib/task.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
```

Comment:
As per our discussion, I think this should probably be "scheduler" and should import the scheduler from the animation layer.

You may wish to do this as a separate CL (maybe before this one, maybe after, whichever is easiest) since that CL will involve a lot of changes to other files.